### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kaneriyapoojan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ supports both Linux and Windows virtual environments.
 Your support counts!
 --------------------
 
-Qubes OS is free and open-source software. If you rely on Qubes for secure
+Qubes OS is a free and open-source software. If you rely on Qubes for secure
```
Comment (andrewdavidwong) [inline README.md:17]: This is ungrammatical.

File: README.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ Deprecated Documentation
 ------------------------
 
  - [qubes-os.org-3.2-EOL.zip] contains the state of the Qubes website when
-   [Qubes OS 3.2 reached EOL]. All you need to browse it is a web browser.
+   [Qubes OS 3.2 reached EOL]. All you need to browse is a web browser.
```
Comment (andrewdavidwong) [inline README.md:133]: This is fine but not necessary. Both sentences are fine.

**Comment by woju** [comment]
@andrewdavidwong this looks like @hacktoberfest and it's not even October yet. Brace for a month of spammy submissions.
https://blog.domenic.me/hacktoberfest/

Can you mark this and as "invalid"? If this tweet: https://mobile.twitter.com/MattIPv4/status/1311366041897971712 is to be believed, "invalid" won't count and might deter spammers.

**Comment by andrewdavidwong** [comment]
> this looks like @Hacktoberfest and it's not even October yet. Brace for a month of spammy submissions.
https://blog.domenic.me/hacktoberfest/
> 
> Can you mark this and as "invalid"? If this tweet: https://mobile.twitter.com/MattIPv4/status/1311366041897971712 is to be believed, "invalid" won't count and might deter spammers.

Yes, I received a response from GitHub Support indicating the same.

**Pull request closed without merging**